src/tbb/rml_thread_monitor.h

```diff
@@ -183,6 +184,33 @@ inline void thread_monitor::check( int error_code, const char* routine ) {
     }
 }
 
+// pthread_create(2) can spuriously fail on Linux. This is a function
+// to wrap pthread_create(2) to retry if it fails with EAGAIN.
+inline void do_pthread_create(pthread_t *handle, pthread_attr_t *s, void* (*thread_routine)(void*), void* arg) {
+#ifdef __linux__
```

Comment:
Let's use it as default implementation for system that use pthread_create, so please remove macro

Reply:
Done